src/convertToObject.js

```diff
@@ -5,8 +5,28 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const getDeclarationAsStrings = sourceString
+    .split(';')
+    .map((rawDeclaration) => rawDeclaration.trim());
+  const getArrayOfDeclarationEntries = getDeclarationAsStrings.map(
+    (preparedDeclaration) =>
+      preparedDeclaration
+        .split(':')
+        .map((preparedDeclarationToTrim) => preparedDeclarationToTrim.trim()),
+  );
+
+  return getArrayOfDeclarationEntries.reduce((acc, declarationEntry) => {
+    if (declarationEntry.length === 1) {
+      return acc;
+    }
+
+    return {
+      ...acc,
+      [declarationEntry[0]]: declarationEntry[1],
+    };
+  }, {});
```

Comment:
The function `convertToObject` does not actually exist in the provided code snippet. Make sure to define the function and its logic according to the task requirements.